Tests/FixtureIntegrationTest.php

```diff
@@ -41,7 +41,7 @@ protected function setUp()
 
     public function testFixturesLoader() : void
     {
-        $kernel = new IntegrationTestKernel('dev', true);
+        $kernel = new IntegrationTestKernel('dev', false);
```

Comment:
Is this pertinent to the PR, or just some extra cleanup?

Reply:
This was some necessary cleanup as tests started failing.